installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3020,6 +3020,19 @@ CREATE TABLE IF NOT EXISTS `cf_AssetGroupCriteriaDetails` (
   PRIMARY KEY (`id_`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
 
+CREATE TABLE IF NOT EXISTS `cf_outbound_accounts` (
```

Comment:
can we use the same cf_accounts table? because all the columns are same

Reply:
@ershad-paladin during design discussion phase I and @santhosh-challa discussed and brainstormed on this. There we decided to have seperate table for outbound plugin to avoid testing efforts whenever existing code coupled with cf_accounts undergo any change in future. Also we want to have clear demarcation between inbound and outbound plugins for better maintainability to avoid tight coupling. For instance assets/violations column does not make much sense for outbound plugin, but i maintained them in table structure just for combining them using UNION ALL to fetch list plugin API. 
CC @santhosh-challa  @akashpaladin

Reply:
If we added assets and violations column just as a workaround for the union query, I'd suggest to remove the columns with below change in API
"UNION ALL \n" +
                "SELECT \n" +
                "       o.accountId, \n" +
                "       o.accountName, \n" +
                "       0 as assets, \n" +
                "       0 as violations, \n" +
                "       o.accountStatus, \n" +
                "       o.platform, \n" +
                "       o.createdBy, \n" +
                "       o.createdTime \n" +
                "FROM cf_outbound_accounts o) AS plugins";

Reply:
addressed in latest push Modified List API query as well as DDL statement.